steps/android-info/1.0.1/step.yml

```diff
@@ -0,0 +1,51 @@
+title: Android info
+summary: Android APK info provider
+description: Provides all possible Android APK information as package name, version
+  name or version code.
+website: https://github.com/thefuntasty/bitrise-step-android-info
+source_code_url: https://github.com/thefuntasty/bitrise-step-android-info
+support_url: https://github.com/thefuntasty/bitrise-step-android-info/issues
+published_at: 2016-10-19T14:14:42.059481445+02:00
+source:
+  git: https://github.com/thefuntasty/bitrise-step-android-info.git
+  commit: 9d343a42df2b8a3b64ec328b236f1fc609aaf0d8
+project_type_tags:
+- android
+type_tags:
+- artifact
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: false
+run_if: .IsCI
```

Comment:
`run_if: .IsCI` means the step should run __only__ if `bitrise-cli` runs in CI mode (for example on website), so locally the step will be skipped (if CI mode not set by hand).